### PR TITLE
[designspaceLib] Document proposed public.skipExportGlyphs lib key

### PR DESCRIPTION
**Pull request by madig**
UFO specification discussion: https://github.com/unified-font-object/ufo-spec/issues/68
UFO specification PR: https://github.com/unified-font-object/ufo-spec/pull/84
ufo2ft PR: https://github.com/googlei18n/ufo2ft/pull/318

### CODE REVIEW

File: Doc/source/designspaceLib/readme.rst [hunk 1]
```diff
@@ -1044,6 +1044,19 @@ Recommendation for editors
 -  Use ``documentObject.updateFilenameFromPath()`` to explicitly set the
    **filename** attributes for all instance and source descriptors.
 
+Common Lib Key Registry
```
Comment (benkiel) [inline Doc/source/designspaceLib/readme.rst]: Formatting niggles, I think this should bump up to the level of a heading, be given the number 7, and have the `7 This document` change to `8 This document`

**Comment by benkiel** [comment]
@madig I think I made those edits correctly, but now I'm not seeing the rendered rst preview, so please take a look and see what you think.

**Comment by madig** [comment]
I think you have to write

```
public.skipExportGlyphs
-----------------------
```

Otherwise, I get an error when pasting the text on http://rst.ninjs.org.

**Comment by benkiel** [comment]
@madig Got it, thanks. Sigh, rst. This looks good to me, will merge.

**Comment by madig** [comment]
I can't approve my own PR :smile:

**Pull request merged**